.github/workflows/jre11-scan.yml

```diff
@@ -0,0 +1,27 @@
+name: Scan jre/11 Docker image
```

Comment:
This file is copied and revised from https://github.com/scalar-labs/docker/blob/main/.github/workflows/jdk8-scan.yml